livesql/live.go

```diff
@@ -111,17 +113,30 @@ type queryCacheKey struct {
 
 // query reactively performs a SelectQuery
 func (ldb *LiveDB) query(ctx context.Context, query *sqlgen.SelectQuery, filter sqlgen.Filter) ([]interface{}, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.livesql.query")
+	fromCache := true
+	defer func() {
+		span.LogFields(
```

Comment:
I think these (and the other calls to `LogFields`) would look nicer on just a single line.